lib/views/branch-list-view.coffee

```diff
@@ -9,6 +9,7 @@ class ListView extends SelectListView
 
   initialize: (@repo, @data) ->
     super
+    @addClass('git-branch')
```

Comment:
> Is there styling that uses this class name?
Yeah, a theme I am working on for one, which adds icons to the modals. Packages often don’t bother to add a class to them, with the result being the inability to target them individually for styling.

With this line the class selector becomes `.select-list.git-branch` or `atom-panel .modal .select-list.git-branch` to be more specific, so shouldn’t be a problem unless I’m missing something? (Working fine on my local fork…)